variables.tf

```
@@ -66,59 +40,101 @@ variable "storage_class_name" {
 variable "prometheus_resources" {
   type        = any
```

I would prefer to define the object structure using the following syntax:

```suggestion
  type        = object({ cpu_requests = string, memory_requests = string })
```